src/components/organisms/NewResourceWizard/NewResourceWizard.tsx

```diff
@@ -150,7 +146,7 @@ const NewResourceWizard = () => {
     } else {
       selectedFolderResources = Object.values(resourceMap).filter(
         resource =>
-          resource.filePath.split(path.sep).length > 2 && getDirPath(resource.filePath).endsWith(selectedFolder)
+          resource.filePath.split(path.sep).length > 2 && path.dirname(resource.filePath).endsWith(selectedFolder)
```

Comment:
We should use `path.dirname` depending on the os platform;

```
const osPlatform = useAppSelector(state => state.config.osPlatform);

const getDirname = osPlatform === 'win32' ? path.win32.dirname : path.dirname;.

...getDirname(resource.filePath).endsWith(selectedFolder);
```

Reply:
My bad @topliceanurazvan I was modifying this only!

Reply:
Please check now